composer/workflows/dataflow_template_operator_tutorial/create_dataset_and_table_test.py

```diff
@@ -0,0 +1,66 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
+
+client = bigquery.Client()
+dataset_UUID = str(uuid.uuid4())
+dataset_id = "{}.sample_dataset_{}".format(client.project, dataset_UUID) 
```

Comment:
[f-strings](https://www.python.org/dev/peps/pep-0498/) are preferred now instead of .format

Reply:
TIL
Sorry about advocating `str.format()`. My python knowledge was around python-2.7 :)